runtimes/dart-2.13/appwrite_function_types/lib/appwrite_function_types.dart

```diff
@@ -0,0 +1,33 @@
+import 'dart:convert';
```

Comment:
These function types for dart and type definition for other languages should be published, as developers will have to use it in their code. Let's discuss this.